types/geojson/index.d.ts

```diff
@@ -70,67 +76,80 @@ export interface GeometryObject extends GeoJsonObject {
     type: GeoJsonGeometryTypes;
 }
 
+/**
+ * Union of geometry objects.
+ * https://tools.ietf.org/html/rfc7946#section-3
+ */
+export type Geometry = Point | MultiPoint | LineString | Polygon | MultiPolygon | GeometryCollection;
```

Comment:
Similar circumstances here (like described above). All elements in your `Geometry` type are extended from `GeometryObject`. So I don't see the advantage of a type `Geometry` here. Just use something like: `any extends GeometryObject`.

Reply:
They're significantly more powerful, see my examples below.